src/physics/physics.cpp

```diff
@@ -77,9 +87,15 @@ RayHit collision_line_box(Vec3 origin, Vec3 dir, Box a) {
     }
 
     // Ray origin inside bounding box
-    if (inside) {
+    //
+    // NOTE(ed): Not returning a normal here
+    // makes things more numerically unstable.
+    // I didn't mange to think of a way to do this,
+    // but there might be a smart way to do it here.
+    // Otherwise we fall back on the "collision_check"
+    // function for the "t" we know they are overlapping.
```

Comment:
```suggestion
    // Otherwise we fall back on the "collision_check"
    // function for the "t" since we know they are overlapping.
```
? If I understood the comment at least.